### PR TITLE
Add playwright nodejs module

### PR DESCRIPTION
**Pull request by javierlopezdeancos**
# Add Playwright community module to nodejs

Add a new community module to [`Playwright`](https://playwright.dev) to` testcontaines-node`

### CODE REVIEW

**Comment by cristianrgreco** [comment]
Playwright module for Testcontainers node doesn't exist

File: modules/playwright/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+categories:
+  - web
+docs:
+  - id: nodejs
+    url: https://github.com/javierlopezdeancos/testcontainers-node-playwright
+    maintainer: community
+    example: |
+      ```javascript
+      import path from "path";
+      import { PlaywrightContainer } from "@testcontainers/playwright";
```
Comment (cristianrgreco) [inline modules/playwright/index.md:12]: testcontainers-node-playwright?

Comment (javierlopezdeancos) [reply modules/playwright/index.md:12]: Ups, thanks to your feedback @cristianrgreco 
As usual, you are right 
https://github.com/javierlopezdeancos/testcontainers-node-playwright/blob/main/package.json#L2
https://www.npmjs.com/package/testcontainers-node-playwright

**Review by kiview — APPROVED** [review]

**Comment by kiview** [comment]
Thanks @javierlopezdeancos 🙏

**Pull request merged**